var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -76,6 +76,8 @@ def install(self, spec, prefix):
 
         if '+libiberty' in spec:
             configure_args.append('--enable-install-libiberty')
+            configure_args.append('--disable-nls')
```

Comment:
Well, not quite actually. The `libiberty` variant is not built by default, so this would in fact have no effect when the default build settings are used.